locale/en.lua

```diff
@@ -120,7 +120,7 @@ local Translations = {
     },
 }
 
-Lang = Lang or Locale:new({
+Lang = Locale:new({
```

Comment:
```suggestion
Lang = Lang or Locale:new({
```
Whoops... This file *will* be loaded twice (`locale/en.lua` & `locale/*`), so this should be added back to make sure that the selected language will not be replaced by English.

Reply:
I removed it from the fxmanifest as well, so you might want to make a review on that